### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MattWestb**
The ZB3 section i think can being deleted and being adopted to radio modules that is /can running ZHA1.X.
I is missing one short part describing ZHA-Quirks that is one large part of how ZHA / Zigpy is working.

Feel free closing / deleting is not liking it !!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,39 +13,37 @@ zigpy contains common code implementing Zigbee ZCL, ZDO and application state ma
...
 
-### Zigbee device OTA updates
-
-zigpy have code to automatically download and perform OTA (Over-The-Air) firmware updates of Zigbee devices if the OTA firmware image provider source URL for updates is known. The OTA update directory is optional and it can also be used for any offline firmware files that you provide yourself.
-
-Online OTA providers for firmware updates are currently only available for IKEA and LEDVANCE devices. OTA updates for device of other manufactures could possible also be supported by zigpy in the future, if these manufacturers publish their device OTA firmware images publicly.
+## Compatible Zigbee coordinator hardware
 
-## How to install and test, report bugs, or contribute to this project
+Radio libraries for zigpy are separate projects with their own repositories and implanting the support for the hardware radios modules.
```
Comment (Adminiuga) [inline README.md:18]:
```suggestion
Radio libraries for zigpy are separate projects with their own repositories and implementation for the radios modules.
```

File: README.md [hunk 2]
```diff
@@ -13,39 +13,37 @@ zigpy contains common code implementing Zigbee ZCL, ZDO and application state ma
...
+Stabile compatible radio labaries:
```
Comment (Adminiuga) [inline README.md:19]:
```suggestion
Stable radio labaries: 
```

Comment (puddly) [reply README.md:19]: `labaries` -> `libraries` ?

Comment (Adminiuga) [reply README.md:19]: this whole thing needs to be ran through grammarly.

**Review by Adminiuga — CHANGES_REQUESTED** [review]
Please run it through grammarly.

**Pull request closed without merging**